packages/core/src/awsService/ec2/sshKeyPair.ts

```diff
@@ -5,19 +5,32 @@
 import * as fs from 'fs-extra'
```

Comment:
If it's not much trouble, it would be helpful to remove this and use our fs.ts module instead. We want to eliminate 'fs-extra'

Reply:
I couldn't figure out how to set file permission (`fs.chmod` equivalent) through `fs.ts` but it appears redundant as ssh-keygen was already doing so properly so believe we're all good.

Reply:
> how to set file permission (`fs.chmod` equivalent) through `fs.ts`

would you mind adding a stub for chmod in fs.ts? it can just do nothing for `if (isWeb())`. and for non-web, it can use [node's chmod](https://nodejs.org/api/fs.html#fspromiseschmodpath-mode) (from 'fs', instead of 'fs-extra)